docs/using/backup.md

```diff
@@ -0,0 +1,98 @@
+# Backup and Restore
+
+> <b style="color:goldenrod">Caution</b> Backup and restore features are a technical preview and currently only work with MySQL database servers backing up to Amazon AWS S3 storage. locations
+
+## Before you start
+
+**For restoring**
+
+There are no special requirements for restoring.
```

Comment:
Aren't the requirements same as for backing up? i.e., you gotta have access to S3, connectivity, the service has be there, etc. Well, probably a bit different, but still...

Reply:
Yes I realise I misinterpreted some facts. Restoring can be done without installing xtrabackup et al, but sure, still needs the BU location and a server to put it. I'll amend that.